infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/devfile/DockerimageComponentToWorkspaceApplier.java

```diff
@@ -120,10 +111,37 @@ public void apply(
     String machineName =
         componentAlias == null ? toMachineName(dockerimageComponent.getImage()) : componentAlias;
 
+    MachineConfigImpl machineConfig = createMachineConfig(dockerimageComponent, componentAlias);
+    List<HasMetadata> componentObjects = createComponentObjects(dockerimageComponent, machineName);
+
+    k8sEnvProvisioner.provision(
+        workspaceConfig,
+        KubernetesEnvironment.TYPE,
+        componentObjects,
+        ImmutableMap.of(machineName, machineConfig));
+
+    workspaceConfig
+        .getCommands()
+        .stream()
+        .filter(
+            c ->
+                componentAlias != null
```

Comment:
is that possible that `Commands` have `null` in `Attributes`

Reply:
hopefully not, I didn't touched this code, just github diff marked it as + because I've shuffled the code a bit.